### PR TITLE
Port ACE2 burning

### PR DESCRIPTION
**Pull request by commy2**
**When merged this pull request will:**
- Ports ACE2 "fire propagation" component
- requires: https://github.com/CBATeam/CBA_A3/pull/441
- [ ] screem and panic

### CODE REVIEW

File: addons/fire/functions/fnc_burn.sqf [hunk 1]
```diff
@@ -0,0 +1,245 @@
...
+
+        // add damage
+        private _damage = damage _unit + FIRE_DPS * _timeStep;
+        private _damages = getAllHitPointsDamage _unit param [2, []] apply {_x + FIRE_DPS * _timeStep};
+
+        _unit setDamage _damage;
+
+        {
+            _unit setHitIndex [_forEachIndex, _x];
+        } forEach _damages;
```
Comment (Glowbal) [inline addons/fire/functions/fnc_burn.sqf]: damage handling by passes the medical system. Should call the medical damage API using `fire` as the projectile type.

Comment (commy2) [inline addons/fire/functions/fnc_burn.sqf]: rgr

**Comment by jonpas** [comment]
https://github.com/CBATeam/CBA_A3/pull/441 has been released for a while. Is this PR still on the list or will this be integrated into medical-rewrite? @commy2

**Comment by bux** [comment]
@commy2 Status?

**Comment by thojkooi** [comment]
We will pick this up after the medical rewrite.

**Comment by jonpas** [comment]
Deprecated by #7565.

**Pull request closed without merging**